### PR TITLE
JAMES-3078 Reactive Set/Get JMAP filters

### PR DESCRIPTION
**Pull request by chibenwa**
Based on https://github.com/linagora/james-project/pull/3241

### CODE REVIEW

**Comment by chibenwa** [comment]
```
[f9c01f9c66dc6d3bc144d591d06df017d47e566c] [ERROR]   MemoryFilterTest>FilterTest.messageShouldBeAppendedInInboxWhenExactlyEqualsComparatorDoesNotMatch:1722 1 expectation failed.
[f9c01f9c66dc6d3bc144d591d06df017d47e566c] Expected status code <200> but was <500>.
[f9c01f9c66dc6d3bc144d591d06df017d47e566c] 
[f9c01f9c66dc6d3bc144d591d06df017d47e566c] [ERROR]   MemoryFilterTest>FilterTest.messageShouldBeAppendedInInboxWhenNotExactlyEqualsComparatorMatchesName:2052 1 expectation failed.
[f9c01f9c66dc6d3bc144d591d06df017d47e566c] Expected status code <200> but was <500>.
```

File: dockerfiles/run/spring/Dockerfile [hunk 1]
```diff
@@ -17,7 +17,7 @@ EXPOSE 25 110 143 465 587 993 4000
 
 WORKDIR /root
 
-ARG VERSION=3.5.0-SNAPSHOT
+ARG VERSION=3.6.0-SNAPSHOT
```
Comment (Arsnael) [inline dockerfiles/run/spring/Dockerfile:20]: Why do you have a release commit in this PR???

Comment (chibenwa) [reply dockerfiles/run/spring/Dockerfile:20]: That's required for fixing the build and it will disappear with a rebase on master

**Comment by chibenwa** [comment]
test this please

**Comment by chibenwa** [comment]
Merged

**Pull request closed without merging**